docs/content/configuration.md

```diff
@@ -102,6 +102,14 @@ these are rarely used for various reasons.
 | `sessionLife`         | `CMD_SESSION_LIFE`          | **`14 * 24 * 60 * 60 * 1000`**, `1209600000` (14 days)                  | Cookie session life time in milliseconds.                                                                                                                      |
 | `sessionSecret`       | `CMD_SESSION_SECRET`        | **`secret`**                                                            | Cookie session secret used to sign the session cookie. If none is set, one will randomly generated on each startup, meaning all your users will be logged out. |
 
+## Guest Notes autolock
+
+| config file           | environment                 | **default** and example value                                           | description                                                                                                                                                    |
+| --------------------- | --------------------------- | ----------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `lockGuestNotes` | | `{ timeout: 0,  permission: 'locked' }` | If the `timeout` setting is `>` 0, the autolock feature is activated. It'll automatically set the permission of guest notes, that exist longer than defined in the timeout setting, to the permission defined in the `permission` setting. |
```

Comment:
these are missing what the base of the timeout is. It should be noted that the timeout starts from creation, not modification date.

Reply:
okay, maybe the "that exist longer than defined in the timeout setting" didn't make clear, that creation date is used. I'll find a better explanation ^^